### PR TITLE
Secure endpoints to create, edit, and delete announcements

### PR DESCRIPTION
**Pull request by seancorc**
## Overview

  
Add authentication to the endpoints for creating, editing, and deleting announcements.
 
 
  ## Changes Made

  
Created an 'authenticate' decorator based off the one found [here](https://github.com/cuappdev/archives/blob/master/register/src/app/eventregister/s/utils/authorize.pyl)

 
 
  ## Test Coverage

  
Tested using postman.
 
 
  ## Next Steps 

  
Creating documentation.
 
 
  ## Related PRs or Issues 

  
[Issue #6](https://github.com/cuappdev/announcements-backend/issues/6)

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 INVALID_REQUEST_BODY_ERROR = json.dumps({"success": False, "error": "Invalid request body"})
 SUCCESSFUL_RESPONSE = json.dumps({"success": True})
 INVALID_APP_NAME_ERROR = json.dumps({"success": False, "error": "Invalid app identifier(s)"})
+MISSING_REQUEST_TOKEN_ERROR = json.dumps({"success": False, "error": "Missing access token"})
+INVALID_REQUEST_TOKEN_ERROR = json.dumps({"success": False, "error": "Invalid access token"})
```
Comment (kq-li) [inline constants.py:7]: alphabetize these

**Review by kq-li — APPROVED** [review]
clean

**Review by meganle — APPROVED** [review]
very clean!

**Review by manishsaha — APPROVED** [review]
v neat

**Pull request merged**